### DISCUSSION TITLE
I can't get any points from joining discord and telegram group

### DISCUSSION THREAD

**Discussion by Elrondtwinblade** [IBAX-io/go-ibax#776] [Q&A] [answered]
### Describe the bug

I can't get any points from joining discord and telegram group. Can someone solve this problem? 

Discord ve telegram grubuna katıldığımda puan kazanamadım aynı sorunu yaşayan başkaları var mı acaba?

### Reproduction

https://ibax.io/

### System Info

```shell
https://ibax.io/
```

### Validations

- [X] Read the [docs](https://docs.ibax.io).
- [X] Check that there isn't [already an issue](https://github.com/IBAX-io/go-ibax/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/IBAX-io/go-ibax/discussions) or join our [Discord Chat Server](https://discord.com/invite/zRX6Mwafya).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by HalloKyle** [comment, marked as answer]
You can participate the community activities and get IBAX points when you join the Telegram/Discord group.
